packages/manager/core/generator/app/templates/package.json.hbs

```diff
@@ -28,8 +28,8 @@
     "@ovh-ux/request-tagger": "*",
     "@tanstack/react-query": "^5.51.21",
     "@tanstack/react-query-devtools": "^5.51.21",
-    "@ovhcloud/ods-components": "18.0.0",
-    "@ovhcloud/ods-themes": "18.0.0",
+    "@ovhcloud/ods-components": "^18.3.0",
+    "@ovhcloud/ods-themes": "^18.3.0",
     "@ovh-ux/manager-react-components": "2.0.0",
```

Comment:
You also need to bump `manager-react-components` to use latest version.

Reply:
👍